CRM/Core/BAO/File.php

```diff
@@ -824,6 +824,8 @@ public static function getEntityTables(): array {
       'civicrm_activity' => ts('Activity'),
       'civicrm_case' => ts('Case'),
       'civicrm_note' => ts('Note'),
+      'civicrm_search_display' => ts('Search Display'),
+      'civicrm_saved_search' => ts('Saved Search'),
```

Comment:
I'd prefer attaching the file consistently to one or the other. It would be more predictable.